dask/array/core.py

```diff
@@ -2866,9 +2881,24 @@ def from_array(
     if lock is True:
         lock = SerializableLock()
 
+    limit = config.get("array.chunk-size")
+    if isinstance(limit, str):
+        limit = parse_bytes(limit)
+    is_ndarray = type(x) is np.ndarray
+    array_too_big = is_ndarray and x.nbytes > limit
```

Comment:
Are there cases where the array is not too big, but there are many blocks, where we would not want to slice eagerly?

Reply:
hmmm... i'm not sure how to judge this.

Reply:
I'll convert my question into a suggestion:

I recommend that we just always eagerly slice numpy arrays.  I can not think of an example where this would not be welcome.

Reply:
Sounds good.